modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace/collection.json

```diff
@@ -0,0 +1,4 @@
+{
+	"description": "Marketplace",
```

Comment:
@jotchon  What was your thinking about this? I was thinking we'd keep the public site nav fragment exactly the same in each place so it's easier to update

Reply:
yeah you're right.  We should keep it the same for all since we can copy and paste it easier.

Reply:
Fixed with #8